verifyReleaseResources fails during release build

I get an error when building the android APK

```
> Task :react-native-intent-launcher:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-intent-launcher:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  error: resource android:style/TextAppearance.Material.Widget.Button.Borderless.Colored not found.
  error: resource android:style/TextAppearance.Material.Widget.Button.Colored not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:7: error: resource android:attr/colorError not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:11: error: resource android:attr/colorError not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:15: error: style attribute 'android:attr/keyboardNavigationCluster' not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2711: error: resource android:attr/fontStyle not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2712: error: resource android:attr/font not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2713: error: resource android:attr/fontWeight not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2714: error: resource android:attr/fontVariationSettings not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2715: error: resource android:attr/ttcIndex not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2796: error: resource android:attr/startX not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2799: error: resource android:attr/startY not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2802: error: resource android:attr/endX not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2805: error: resource android:attr/endY not found.
  /home/dara/git/test/node_modules/react-native-intent-launcher/android/build/intermediates/res/merged/release/values/values.xml:2813: error: resource android:attr/offset not found.
  error: failed linking references.
```

To reproduce:

```bash
react-native init test
cd test
yarn add react-native-intent-launcher
react-native link
cd android
./gradlew assembleRelease
```

Any ideas?

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -5,7 +5,7 @@ def safeExtGet(prop, fallback) {
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion safeExtGet("compileSdkVersion", 28.0.2)
+    compileSdkVersion safeExtGet("compileSdkVersion", "28.0.2")
     buildToolsVersion safeExtGet("buildToolsVersion", 28)
 
     defaultConfig {

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,12 +1,15 @@
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion safeExtGet("compileSdkVersion", 28.0.2)
+    buildToolsVersion safeExtGet("buildToolsVersion", 28)
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
+        minSdkVersion safeExtGet('minSdkVersion', 16)
         versionCode 1
         versionName "1.0"
     }
@@ -14,6 +17,6 @@ android {
 
 dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
+    compile "com.android.support:appcompat-v7:+"
     compile "com.facebook.react:react-native:+"  // From node_modules
 }

--- a/android/src/main/java/com/poberwong/launcher/IntentLauncherModule.java
+++ b/android/src/main/java/com/poberwong/launcher/IntentLauncherModule.java
@@ -60,11 +60,17 @@ public class IntentLauncherModule extends ReactContextBaseJavaModule implements 
         if (params.hasKey(ATTR_ACTION)) {
             intent.setAction(params.getString(ATTR_ACTION));
         }
-        if (params.hasKey(ATTR_DATA)) {
-            intent.setData(Uri.parse(params.getString(ATTR_DATA)));
-        }
-        if (params.hasKey(ATTR_TYPE)) {
-            intent.setType(params.getString(ATTR_TYPE));
+        // setting data resets type; and setting type resets data; if you have both, you need to set them at the same time
+        // https://developer.android.com/guide/components/intents-filters.html#Types (see 'Data' section)
+        if (params.hasKey(ATTR_DATA) && params.hasKey(ATTR_TYPE)) {
+            intent.setDataAndType(Uri.parse(params.getString(ATTR_DATA)), params.getString(ATTR_TYPE));
+        } else {
+            if (params.hasKey(ATTR_DATA)) {
+                intent.setData(Uri.parse(params.getString(ATTR_DATA)));
+            }
+            if (params.hasKey(ATTR_TYPE)) {
+                intent.setType(params.getString(ATTR_TYPE));
+            }
         }
         if (params.hasKey(TAG_EXTRA)) {
             intent.putExtras(Arguments.toBundle(params.getMap(TAG_EXTRA)));

